subprojects/plugins/src/main/java/org/gradle/api/tasks/Upload.java

```diff
@@ -139,6 +139,7 @@ public RepositoryHandler repositories(Action<? super RepositoryHandler> configur
      *
      * @return the artifacts.
      */
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Do we care about paths here?

Reply:
Changed to `NAME_ONLY`. The artifacts are identified by the name.